Dapper.Tests/Dapper.Tests.csproj

```diff
@@ -6,38 +6,36 @@
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
-    <TargetFrameworks>net452;netcoreapp1.0</TargetFrameworks>
+    <TargetFrameworks>net452;netcoreapp1.0;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.0</TargetFrameworks>

```

Comment:
Nope. Good catch! I was proving out a bug in xUnit beta 4 and forgot to remove - will do so after this set of cleanup commits.